src/groups/mqb/mqbnet/mqbnet_tcpsessionfactory.h

```diff
@@ -97,6 +97,7 @@
 // BDE
 #include <bdlbb_blob.h>
 #include <bdlmt_eventscheduler.h>
+#include <bdlpcre_regex.h>
```

Comment:
Would ideally avoid exposing this include in the header, since it will go to all other source files.

Reply:
Ok, I can move this class (or its port extracting functionality as a separate class) to the source file. And create a static instance there.

Reply:
agreed to leave it as it is